consensus/src/twins/basic_twins_test.rs

```diff
@@ -25,6 +25,7 @@ use std::collections::HashMap;
 ///
 /// Run the test:
 /// cargo xtest -p consensus basic_start_test -- --nocapture
+#[ignore]
```

Comment:
This can be reverted after rebase on #6978

Reply:
I actually did this in this pr